lib/supa/commands/attributes.rb

```diff
@@ -0,0 +1,18 @@
+module Supa
+  module Commands
+    class Attributes < Supa::Command
```

Comment:
I don't see a reason why you change interface, please don't do that, you break Open-Close principle.
it doesn't make so much sense `name` it or `names`